.github/workflows/_rust.yml

```diff
@@ -65,7 +65,7 @@ jobs:
         id: setup-rust
       - uses: ./.github/actions/setup-tauri
         timeout-minutes: 5
-      - run: cargo test --all-features ${{ steps.setup-rust.outputs.packages }} -- --include-ignored
+      - run: cargo test --release --all-features ${{ steps.setup-rust.outputs.packages }} -- --include-ignored
```

Comment:
We already override with optimisation level 1 further down. Making a release build (or setting the opt-level to 3) disables debug assertions which we can't do for the unit tests.

We'll need to move this out of the unit tests in that case.